src/pysofar/spotter.py

```diff
@@ -217,25 +217,42 @@ def update(self):
         }
         self._data = results
 
-    def latest_data(self, include_wind: bool = False, include_directional_moments: bool = False):
+    def latest_data(self, 
+                    include_wind: bool = False, 
+                    include_directional_moments: bool = False,
+                    include_barometer_data: bool = False,
+                    include_partition_data: bool = False,
+                    include_surface_temp_data: bool = False):
         """
 
         :param include_wind:
```

Comment:
Probably good to add descriptions for input here

Reply:
OK, pushed a new commit with the comments requested